src/devices/Ssd1306/Command/ICommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ssd1306.Command
+{
+    public interface ICommand
+    {
+        byte Value { get; }
+
+        byte[] GetBytes();
```

Comment:
If we do end up changing the way we get the bytes of the command in order to use Span and cheaper operations, we should probably expose the Length of bytes from this interface as well.